src/metrics/postgres_metric.ts

```diff
@@ -0,0 +1,74 @@
+import { AbstractMetric, Datasource, MetricId, MetricQuery } from './metric';
+
+import * as _ from 'lodash';
+export class PostgresMetric extends AbstractMetric {
+
+  private _targetName: string; //save first target name, while multi metric not implemented
+
+  constructor(datasource: Datasource, targets: any[], id?: MetricId) {
+    super(datasource, targets, id);
+    this._targetName = targets[0].refId;
+  }
+
+  getQuery(from: number, to: number, limit: number, offset: number): MetricQuery {
+    let queries = this.datasource['data']['queries'];
+    _.forEach(queries, (q) => {
+        q.rawSql = this.processLimitOffset(q.rawSql, limit, offset);
+    });
+    return {
+      url: this.datasource.url,
+      method: 'POST',
+      schema: {
+        data: {
+          from: `${from}`,
+          to: `${to}`,
```

Comment:
Could be just
```javascript
...
data: {
  from,
  to,
  queries
}
...
```

Reply:
for correct request it is necessary cast `from` and `to` to string. May be there is better way to do this?